src/stdatm/speed_parameters.py

```diff
@@ -10,3 +11,249 @@
+
+
+# From EAS -----------------------------------------------------------------
+def compute_tas_from_eas(equivalent_airspeed, density, sea_level_density):
+    """
+
+    :param equivalent_airspeed: in m/s
+    :param density: in kg/m**3
+    :param sea_level_density: in kg/m**3
+    :return: true airspeed in m/s
+    """
+    true_airspeed = equivalent_airspeed * np.sqrt(sea_level_density / density)
+    return true_airspeed
+
+
+# From unitary Reynolds -----------------------------------------------------------------
+def compute_tas_from_unit_re(unitary_reynolds, kinematic_viscosity):
+    """
+
+    :param unitary_reynolds: in 1/m
+    :param kinematic_viscosity: in m**2/s
+    :return: true airspeed in m/s
+    """
+    true_airspeed = unitary_reynolds * kinematic_viscosity
+    return true_airspeed
+
+
+# From dynamic pressure -----------------------------------------------------------------
+def compute_tas_from_pdyn(dynamic_pressure, density):
+    """
+
+    :param dynamic_pressure: in Pa
+    :param density: in kg/m**3
+    :return: true airspeed in m/s
+    """
+    true_airspeed = np.sqrt(2.0 * dynamic_pressure / density)
+    return true_airspeed
+
+
+# MACH =================================================================
+def compute_mach(true_airspeed, speed_of_sound):
+    """
+
+    :param true_airspeed: in m/s
+    :param speed_of_sound: in m/s
+    :return: Mach number (no unit)
+    """
+    mach = true_airspeed / speed_of_sound
+    return mach
+
+
+# EQUIVALENT AIRSPEED =================================================================
+def compute_equivalent_airspeed(true_airspeed, density, sea_level_density):
+    """
+
+    :param true_airspeed: in m/s
+    :param density: in kg/m**3
+    :param sea_level_density: in kg/m**3
+    :return: equivalent airspeed in m/s
+    """
+    equivalent_airspeed = true_airspeed / np.sqrt(sea_level_density / density)
+    return equivalent_airspeed
+
+
+# UNITARY_REYNOLDS =================================================================
+def compute_unitary_reynolds(true_airspeed, kinematic_viscosity):
+    """
+
+    :param true_airspeed: in m/s
+    :param kinematic_viscosity: in m**2/s
+    :return: unitary_reynolds in 1/m
+    """
+    unitary_reynolds = true_airspeed / kinematic_viscosity
+    return unitary_reynolds
+
+
+# DYNAMIC PRESSURE =================================================================
+def compute_dynamic_pressure(true_airspeed, density):
+    """
+
+    :param true_airspeed: in m/s
+    :param density: in kg/m**3
+    :return: incompressible dynamic pressure in Pa
+    """
+    true_airspeed = 0.5 * density * true_airspeed**2
+    return true_airspeed
+
+
+# IMPACT PRESSURE =================================================================
+def _compute_subsonic_impact_pressure(mach, pressure):
+    return pressure * ((1.0 + 0.2 * mach**2) ** 3.5 - 1.0)
+
+
+COEFF_2 = (GAMMA + 1) / 2 * ((GAMMA + 1) ** 2 / (2 * (GAMMA - 1))) ** (1 / (GAMMA - 1))
```

Comment:
Why is COEFF_2 before COEFF_1 ?

Reply:
The reason is very practical : just because COEF_1 has been written before 😅 
Indeed, I was not very inspired when choosing the names of these coefficients. I change them in 7a533db373104fb7a7f8025a799822f64c5fd413. Not that more inspired, but at least, there is no more number involved.